RPA.MIT.Notification.Function/Validation/ValidateMessage.cs

```diff
@@ -21,7 +21,10 @@ public static class ValidateMessage
             ""Scheme"": {
                 ""type"": ""string""
             },
-            ""EmailRecipient"": {
+            ""ApproverEmail"": {
```

Comment:
This should still be just EmailRecipient? (Just one generic email, not 2 specific Approver/Creator emails...

Reply:
Reverted the changes